pkg_resources/_vendor/appdirs.py

```diff
@@ -8,10 +8,9 @@
 See <http://github.com/ActiveState/appdirs> for details and usage.
 """
 # Dev Notes:
-# - MSDN on where to store app data files:
```

Comment:
Please revert anything in _vendor. These files are maintained upstream. You'll want to contribute this change to the upstream package.